src/main/java/frc/robot/constants/AimingConstants.java

```diff
@@ -70,4 +72,6 @@ public enum AimState {
     public static final int RIGHT_WRIST_ENCODER_ID = 0;
 
     public static final int ELEVATOR_THROUGHBORE_ENCODER_ID = 0;
+    public static final boolean ELEVATOR_LEFT_IS_INVERTED = true;
```

Comment:
In general, it's clearer when you avoid using negatives in symbol names. When you do, it gets confusing when you use those symbols in boolean logic.

So how about switching this to ELEVATOR_LEFT_IS_NORMAL = false

And same for the next symbol.

Reply:
Done.